CHANGES.md

```diff
@@ -1,4 +1,9 @@
 
+## 2.0.0.post1 (2021-02-04)
+
+* fix arguments names conflicts between mosaic_reader/tasks and STACReader options (https://github.com/cogeotiff/rio-tiler/pull/343)
```

Comment:
Tiny nit: I think putting `#343` is enough instead of the full URL

Reply:
last time I tried, it didn't work, maybe it only work when deployed with Mkdocs but I think adding full link has better overall support 🤷‍♂️

Reply:
It should display correctly in Mkdocs: https://facelessuser.github.io/pymdown-extensions/extensions/magiclink/#shorthand-links.

But in the first comment I wasn't even intending that. I was just saying that not-a-link is ok too, but it's really personal preference.